avatar.css

```diff
@@ -0,0 +1,41 @@
+:root{
+    padding: 0;
+    margin: 0;
+    background-color: rgb(24,23,25);
+    font-size: 62.5%;
+    --primary-color: #77A8FD;
+}
+
+.avatar-lg{
+    height: 10rem;
+    width: 10rem;
+    border-radius: 5rem;
+    margin: 1rem;
```

Comment:
**Suggestion**: Here as the **_border-radius_** and **_margin_** are same for all the three classes of lg, md and sm, what you can do is create a **avatar** class and add these there in order to avoid repetition.